javascript/javascript2/week2/main.js

```diff
@@ -38,4 +38,16 @@ function getTheMaximumPrice(e) {
     );
     renderProducts(outcome);
   } 
-}
\ No newline at end of file
+}
+//Add filter by name
+const inputName = document.getElementById("inputName");
+
+let userInputName = inputName.addEventListener("keyup", getTheInputName);
+
+function getTheInputName(e) {
+    const filteredProducts = e.target.value.toLowerCase();
+    const outcome = products.filter((product) => 
+    product.name.toLowerCase().includes (filteredProducts)
+      );
+      renderProducts(outcome);
+  }
```

Comment:
Try to sort product by name a-z and z-a also by low and high price. Again remember to check your task if is running before you are upload it